docs/Testing.rst

```diff
@@ -113,6 +117,16 @@ For every target above, there are variants for different optimizations:
   ``check-swift-non-executable-iphoneos-arm64``) -- runs tests not marked with
   ``executable_test`` in ``-Onone`` mode.
 
+To run the unit tests, they must first be built with the following command
+executed in your build directory:
+
+.. code-block:: bash
+
+    % ninja SwiftUnitTests
```

Comment:
The rest of this document refers to targets without also referencing specific build tools. `SwiftUnitTests` could also be built with `make SwiftUnitTests`, after all.

My suggestion would be to mirror the build invocation for `check-swift-macosx-x86_64` above, and recommend `cmake --build build/Ninja-ReleaseAssert/swift-macosx-x86_64 -- SwiftUnitTests`.